assets/javascripts/discourse/components/group-assignments-filter.js.es6

```diff
@@ -1,5 +1,14 @@
+import discourseComputed from "discourse-common/utils/decorators";
 import Component from "@ember/component";
 
 export default Component.extend({
-  tagName: "li"
+  tagName: "li",
+
+  @discourseComputed("siteSettings.prioritize_username_in_ux", "filter.username", "filter.displayName")
```

Comment:
Where does `filter.displayName` come from? Should that be `filter.name`?

Apart from that question, this is looking great :D

Reply:
filter.name property doesn't value when the user doesn't set name property in the signup, but displayName always have the value